.asf.yaml

```diff
@@ -25,28 +25,28 @@ notifications:
 # https://cwiki.apache.org/confluence/pages/viewpage.action?spaceKey=INFRA&title=Git+-+.asf.yaml+features#Git.asf.yamlfeatures-JenkinsPRwhitelisting
 jenkins:
   github_whitelist:
-    - vcrfxia
     - gharris1727
+    - vcrfxia
     - divijvaidya
     - lucasbru
     - yashmayya
     - philipnee
     - vamossagar12
     - clolov
     - fvaleri
-    - andymg3
+    - akhileshchg
 
 # This list allows you to triage pull requests. It can have a maximum of 10 people.
 # https://cwiki.apache.org/confluence/pages/viewpage.action?spaceKey=INFRA&title=Git+-+.asf.yaml+features#Git.asf.yamlfeatures-AssigningexternalcollaboratorswiththetriageroleonGitHub
 github:
   collaborators:
-    - vcrfxia
     - gharris1727
+    - vcrfxia
     - divijvaidya
     - lucasbru
     - yashmayya
     - philipnee
     - vamossagar12
     - clolov
     - fvaleri
-    - andymg3
+    - akhileshchg
```

Comment:
Was there any decision on expiration date for this rights? If I understand the process now, it might be that `andymg3` won't have the collaborator rights any more. Maybe infra doesn't revoke grants, that I don't know.

Reply:
I have no idea, either. But as the [doc](https://kafka.apache.org/contributing) mentioned:
> The list will be updated as part of the major/minor release process, three to four times a year.